.gitlab/llnl-ci-conda.yml

```diff
@@ -0,0 +1,117 @@
+stages:
+  - env-setup
+  - test
+  - cleanup
+
+.final_steps:
+  script: &finalize
+    - chgrp -fR exaworks $SDK_WORK_DIR/conda || true
+    - chmod -fR 02770 $SDK_WORK_DIR/conda    || true
+
+conda-env-setup:
+  stage: env-setup
+  parallel:
+    matrix:
+      - HOST : [ "quartz", "ruby"]
+        RUNNER_TYPE: "shell"
+        # - HOST : [ "lassen" ]
+        #   ENVIRONMENT: [ "BATCH" ]
+        #   RUNNER_TYPE: "batch"
+  tags:
+    - $HOST
+    - $RUNNER_TYPE
+  script:
+    - mkdir -p ${SDK_WORK_DIR}/conda/
+    - cd ${SDK_WORK_DIR}/conda/
+    - eval "$(conda shell.posix hook)"
+    - conda create --prefix ./${HOST}-exaworks-sdk-venv python=3.7 -y
+    - conda env list
+    - conda activate ./${HOST}-exaworks-sdk-venv
+    - conda env list
+    - ls
+    - pwd 
+    - conda config --add channels conda-forge
+    - conda config --add channels lightsource2-tag 
+    - conda install swift-t
+    - conda install parsl
+    - conda install radical.pilot
+    - pip install pytest
+    - pip install pytest-tap
+    - *finalize
+
+conda-sdk-clone:
+  stage: env-setup
+  needs: [ "conda-env-setup" ]
+  tags:
+    - quartz
+    - shell
+  script:
+    - cd ${SDK_WORK_DIR}/conda/
+    - git clone git@github.com:ExaWorks/SDK.git
+    - *finalize
```

Comment:
this job can be removed, since the config is already part of this repo

Reply:
Done